pulseapi/profiles/serializers/profile_entries.py

```diff
@@ -4,7 +4,7 @@
 from pulseapi.creators.models import EntryCreator
 from pulseapi.entries.models import Entry
 from pulseapi.profiles.models import UserBookmarks
-from pulseapi.entries.serializers import (
+from pulseapi.entries.serializers.base import (
```

Comment:
Since `serializers` is now a directory, I would suggest doing what Chris did for profile serializers in terms of structuring modules (creating an `__init__.py` with an `__all__` property). That way we don't have to make these import changes no matter how we restructure the different modules in the future and can keep the import as `from .serializers import (...)`.